assets/app/features/upgrade/templates/database-configuration-page.jade

```diff
@@ -1,45 +1,83 @@
 .database-configuration-page
     h2(translate='') upgrade.steps.database-configuration.title
+    // TODO(itxaka): temporal crappy solution to show the errors for testing
+    ul.list-group
+        li.list-group-item(ng-repeat='error in upgradeDatabaseVm.errors') {{ error }}
 
-    uib-tabset(active='active')
-        uib-tab(index='0', heading="{{'upgrade.steps.database-configuration.navigation-tabs.create-database' | translate}}")
-            form.database-form.new
+    uib-tabset
+        uib-tab(index='0', disable='upgradeDatabaseVm.running || upgradeDatabaseVm.completed',
+        heading="{{'upgrade.steps.database-configuration.navigation-tabs.create-database' | translate}}")
+            form.database-form.new(name="newForm")
                 .row
                     .col-md-8
-                        .form-group
-                            label(translate='') upgrade.steps.database-configuration.database-form.username
-                            input.form-control(type='text', name='username', ng-required='false', ng-model='upgradeDatabaseVm.databaseForm.username')
+                        .form-group(ng-class="{'has-error': newForm.username.$invalid && newForm.username.$dirty}")
```

Comment:
As an alternative, we could consider using something like: https://www.npmjs.com/package/angular-bootstrap-form-validation. It handles `has-error` internally and in addition provides hints on the cause of validation failure.

Note that this one is not usable directly as it doesn't currently support `ngPattern` and has some old dependencies in `bower.json`. But maybe there are some newer alternatives with similar functionality. To be considered.